### DISCUSSION TITLE
Lidemy 訂閱方案 前端基礎作業設計稿權限

### DISCUSSION THREAD

**Discussion by ecomecom11** [Lidemy/forum#49] [General]
你好老師, 我買了Lidemy訂閱方案, 正在上 [FE101], 本想根據程式導師實驗計畫課綱做HW6 作業, 可是發現設計稿權限僅供公開給第四期學生, 請問能開放權限給Lidemy平台學生嗎? 謝謝

**Comment by aszx87410** [comment]
您好，目前設計稿權限還是僅提供給程式導師實驗計畫的學生喔
如果想要照著做作業的話建議可以參考其他第四期學生寫好的作業，或者是直接參考截圖

**Reply by ecomecom11** [reply]
好的, 謝謝老師回覆
